roles/cockpit/templates/cockpit.yml.j2

```diff
@@ -0,0 +1,25 @@
+http:
+  routers:
+    {{ cockpit_name }}-http:
+      entryPoints:
+        - "web"
+      rule: "Host(`{{ traefik_host }}`)"
+      middlewares:
+        {{ traefik_default_middleware_http.split(',') | to_nice_yaml | trim | indent(8) }}
```

Comment:
No SSO or custom middleware. Why not keep the variable defaults as they were previously, but just reference them here when building the template?

Reply:
I'll have to mull it over a bit. I'm much less familiar with the dynamic traefik files than I should be.

Reply:
There is no need to use that file at all.